lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -106,6 +106,29 @@ function dosomething_user_menu() {
   return $items;
 }
 
+function dosomething_user_menu_alter(&$items) {
+  $items['user/%'] = array(
+    'page callback' => 'dosomething_user_get_view',
+    'page arguments' => array(1),
+    'access callback' => 'user_access',
+    'access arguments' => array('administer modules'),
```

Comment:
@angaither oooo is there a list of all of those somewhere? That one seems very appropriate though